### ISSUE TITLE
fwd backward compatibility version #

### ISSUE THREAD

**Issue by clapierre** [w3c/epub-specs#1112]
This seems odd.  This is version 3.2 yet we are talking about fwd and backward compatibility of 3.0

2.2
Backward Compatibility
It MUST attempt to process an EPUB Publication whose version is lower than "3.0".

Forward Compatibility
It SHOULD attempt to process any given Rendition of an EPUB Publication whose Package Document version attribute designates a version higher than "3.0".

**Comment by dauwhe** [comment]
Charles, the EPUB package file has a version attribute, which is not identical to the version number of the spec. In EPUB 3.0, 3.0.1, and 3.2, the package version attribute value must be 3.0

```xml
<package xmlns="http://www.idpf.org/2007/opf" version="3.0" xml:lang="en" unique-identifier="isbn">
```

**Comment by clapierre** [comment]
Does that need to be stated somewhere may be confusing as I was confused, as the spec # is different from the package version # at least for this area.

**Comment by mattgarrish** [comment]
I think this is ultimately a duplicate of #1111. We should return "Package Document version attribute" to the text.

**Comment by dauwhe** [comment]
Closed via #1113
